### PR TITLE
Geolocation orientation for mobile. Minor fix to pom.xml file

### PR DESCRIPTION
**Pull request by MV88**
## Description
Geolocation orientation for mobile (see #458)

This PR fixes POM xml for the build
## Issues
 -  #458

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)

 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x", remove the other)

 - [ ] Yes
 - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by offtherailz** [comment]
@MV88 how these changes to pom.xml are related with problem to openlayers geolocation tool?

**Comment by MV88** [comment]
> @MV88 how these changes to pom.xml are related with problem to openlayers geolocation tool?

i just wanted to add a light fix in case of a new and fresh env the build was not working
if you prefer i can create another PR

**Comment by offtherailz** [comment]
NP, just it is not related with #458

File: backend/pom.xml [hunk 1]
```diff
@@ -241,10 +241,23 @@
...
-      <repository>
-        <id>geosolutions</id>
-        <name>GeoSolutions Maven Repository</name>
-        <url>http://maven.geo-solutions.it/</url>
-      </repository>
+        <!-- GeoSolutions -->
+        <repository>
+            <id>geosolutions</id>
+            <name>GeoSolutions Repository</name>
+            <url>http://maven.geo-solutions.it</url>
```
Comment (offtherailz) [inline backend/pom.xml:248]:
```suggestion
            <url>https://maven.geo-solutions.it</url>
```

**Comment by MV88** [comment]
> NP, just it is not related with #458

yep, not related

File: backend/pom.xml (hunk 1 above)
Comment (MV88) [reply backend/pom.xml]: mhh i think that also other repos should be fixed then

Comment (MV88) [reply backend/pom.xml]: remember this is aligned to 2021.01.xx

**Comment by offtherailz** [comment]
Now I noticed the change of reference :facepalm:

**Review by offtherailz — APPROVED** [review]

**Pull request merged**